hypervisor/arch/x86/vmx.c

```diff
@@ -606,15 +607,15 @@ static void init_guest_context_vm0_bsp(struct vcpu *vcpu)
 	struct ext_context *ectx =
 		&vcpu->arch_vcpu.contexts[vcpu->arch_vcpu.cur_context].ext_ctx;
 	struct boot_ctx * init_ctx = (struct boot_ctx *)(&vm0_boot_context);
-	uint16_t *sel;
+	uint16_t *sel = &init_ctx->cs_sel;
```

Comment:
we added another rule to use () for  & x->y.   please use &(x->y).

Reply:
Got it. Will fix it in next version.